pom.xml

```diff
@@ -79,6 +79,12 @@
       <version>${folio-spring-support.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-spring-system-user</artifactId>
```

Comment:
By default, the library creates SystemUserService, which is a legacy approach that doesn't work in Eureka. At least the system user default behavior should be overridden (`system-user.enabled = false`),  Ideally, the system user should also be defined.

Reply:
Added definition of system user to module descriptor.